SpatialGDK/Documentation/content/cloud-dev-workflow.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Development Workflow
+
+<!-- This is a live embed of a google drawing -->
+
+<img src="https://docs.google.com/drawings/d/e/2PACX-1vQRmK7TxLji8pT7erPl54hqMMMDsdosZY1OZ2wuPYLQ23dWIrx86qCHggEeq-XasTCsqRe40fCKQvKN/pub?w=758&amp;h=1162">
+
+You may find the below snippets useful as reference:
+
+### Build server-worker assembly
+
+```
+Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat <YourProject>Server Linux Development <YourProject>.uproject
+```
+
+Replacing **`<YourProject>`** with the name of your Unreal project.
+
+### Build client-worker assembly
+
+```
+Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat <YourProject> Win64 Development <YourProject>.uproject
+```
+
+Replacing **`<YourProject>`** with the name of your Unreal project.
+
+### Upload Assembly
+
+```
+spatial cloud upload myassembly
+```
+
+Replacing `myassembly` with the name you choose to give your assembly.
+
+### Launch Cloud Deployment
+
+```
+spatial cloud launch --snapshot=snapshots/default.snapshot <assembly_name> launch_config.json <deployment_name>
+```
+
+Providing:
+
+* `<assembly_name>`, which identifies the worker assemblies to use (as chosen in the `spatial cloud upload` command).
```

Comment:
Who chooses it? Our user? In which case:
```suggestion
* `<myassembly>` -  identifies the worker assemblies to use (as you specified earlier in the `spatial cloud upload` command). Replace `myassembly` with the name you choose to give your assembly.
```
Avoiding "which" means less text to read - nice in a list!  :-)
The explanation says "which assemblies to use" - does that mean you can specify more than one in the command? If not, we should say "assembly to use". If yes, we should say this explicitly.